backend/app/api/src/endpoints/global/members/PatchOrganizationMembersEndpoint.ts

```diff
@@ -410,6 +410,9 @@ export class PatchOrganizationMembersEndpoint extends Endpoint<Params, Query, Bo
             }
 
             // Add platform memberships
+            const organizationTags = organization?.meta.tags ?? [];
+            const memberDefaultAgeGroupIds = member.registrations.map(r => r.group.defaultAgeGroupId).filter(id => id !== null) as string[];
```

Comment:
Niet alle inschrijvingen in member.registrations zijn nog active - deze moeten nog gefilterd worden op basis van periodId en deactivatedAt (moet null zijn) en registeredAt (mag niet null zijn)